puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -60,15 +74,15 @@
   $manage_service         = true,
 ) {
   class { '::cinder::volume':
-    enabled        => str2bool_i("$enabled"),
-    manage_service => str2bool_i("$manage_service"),
+    enabled        => str2bool_i($enabled),
```

Comment:
I am not sure I agree with removing all the double quotes around variables with calling str2bool_i.  We generally try to keep these here as an extra layer of insurance that what is being passed in is indeed a string.